node/db/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 
 mod errors;
 mod memory;
-mod rocks;
+pub mod rocks;
```

Comment:
no need to change this

Reply:
As mentioned, there are compile errors when trying to use the BlockStore trait for Rocksdb without the module being public.

Reply:
Not needed, just tested